src/Asset/SvgImg.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
```

Comment:
Please rename the file to be more specific may be like `SeatIcon` or `SeatComponent`. Since this is a react component, change extension to `jsx` to stay consistent with naming and move it with other react components.